content/developer/howtos/upgrade_custom_db.rst

```diff
@@ -0,0 +1,262 @@
+:show-content:
+
+.. _upgrade_custom/upgrade_custom_db:
+
+===============================
+Upgrading a customized database
+===============================
+
+Upgrading to a new version of Odoo can be challenging, especially if the database you work on
+contains custom modules. This page intent is to explain the technical process of upgrading a
+database with customized modules. For a standard overview, please refer to the
+:doc:`Upgrade documentation </administration/upgrade>`.
+
+We consider a custom module, any module that extends the standard code of Odoo and that was not
+built with the Studio app. 
+Before upgrading those modules, or before asking for them to be upgraded, have a look at the
+:ref:`upgrade/sla` to make sure who's responsible for it.
+
+While working on what we refer to as the **custom upgrade** of your database, keep in mind the goals
+of an upgrade:
+
+#. Stay supported
+#. Get the latest features
+#. Enjoy the performance improvement
+#. Reduce the technical debt
+#. Benefit from security improvements
+
+With every new version of Odoo, changes are introduced. These changes can have an impact modules on
+which customization have been developed. This is the reason why upgrading a database that contains
+custom modules requires additional steps in order to upgrade their source code.
+
+These are the steps to follow to upgrade customized databases:
+
+#. :ref:`Stop the devolopments and challenge them <upgrade_custom/stop_developments>`.
+#. :ref:`upgrade_custom/request_upgrade`.
+#. :ref:`Make your module installable on an empty database <upgrade_custom/empty_database>`.
+#. :ref:`Make your module installable on the upgraded database <upgrade_custom/upgraded_database>`.
+#. :ref:`Test extensively and do a rehearsal <upgrade_custom/testing_rehearsal>`.
+#. :ref:`Upgrade in production <upgrade_custom/production>`.
```

Comment:
If these are really steps you could also prepend numbers to the header.

Reply:
Would you recommend to do something similar to the [Bugfix update page]?(https://www.odoo.com/documentation/17.0/administration/maintain/update.html)
![image](https://github.com/odoo/documentation/assets/28193899/67af70fb-c3a6-40ce-9680-67939535ea43)

Reply:
Yeah, it's an idea. The thing is that very few people actually read documentation through and through; they scroll until they find something that looks like the solution to their problem. Introducing steps will help clarify that sections must be read in a sequence.

Reply:
Sound good to me, I didn't think of that
Thanks for pointing it out! :+1: